changelogs/unreleased/gh-6986-drop-fk-parsing-rules.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/core
```

Comment:
Seems it is mostly about `sql`, so I would suggest `bugfix/sql`. Up to you.

Reply:
Thank you the review! Fixed.